src/actions/Scroll.ts

```diff
@@ -18,12 +17,12 @@ class Scroll implements Action {
       return { lineNumber: getLineNumber(targets, this.at), editor };
     });
 
-    const originalEditor = getActiveTextEditor();
+    const originalEditor = ide().activeEditableTextEditor;
 
     for (const lineWithEditor of lines) {
       // For reveal line to the work we have to have the correct editor focused
-      if (lineWithEditor.editor !== getActiveTextEditor()) {
-        await focusEditor(lineWithEditor.editor);
+      if (lineWithEditor.editor !== ide().activeTextEditor) {
```

Comment:
That entire thing is going to be folded into the editor. For now I have not moved `executeCommand` from the actions into the actual editor. Wants that is done the entire focus thing will be inside the vscode implementation.

Reply:
Fine, but is there any reason not to use `!lineWithEditor.editor.isActive` here?

Reply:
Already are

Reply:
ah right.  looks like my VSCode pull requests extension was being too smart and automatically showing me the source commit because this comment was outdated

Reply:
Ah